packages/trax/trax.0.3.0/descr

```diff
@@ -0,0 +1 @@
+Trax defines a special exception, which is used to store a trace of where the exception was raised and re-raised, independently from the state of the call stack.
```

Comment:
The first line is used as a short description -- would it be possible to have a shorter description of trax on the first line? thanks!